### PR TITLE
Add name email input

### PR DESCRIPTION
**Pull request by veaceslavmocanu**

### CODE REVIEW

File: app/scss/main.scss [hunk 1]
```diff
@@ -48,18 +48,21 @@ body {
...
   background-color: #fff;
   background-clip: padding-box;
-  border-radius: 20px;
+  border-radius: 3px;
 }
 
 .form-group {
   margin-top: 0px;
   margin-right: auto;
   margin-bottom: 1.25rem;
```
Comment (vitaliemiron) [inline app/scss/main.scss]: use only margin inline

File: app/scss/main.scss [hunk 2]
```diff
@@ -48,18 +48,21 @@ body {
 .form-control {
   display: block;
   width: 80%;
-  height: 100%;
-  padding: 5px;
+  height: 40%;
+  padding: 0 0 3px 0;
```
Comment (vitaliemiron) [inline app/scss/main.scss]: if you need only one padding value you can use padding-bottom

File: app/index.html [hunk 3]
```diff
@@ -32,6 +32,11 @@ <h1 id="title" class="text-center main-title">freeCodeCamp Servey Form</h1>
       <label id="email-label" for="Email">Email</label>
       <input id="email" class="form-control" type="text" placeholder="Enter you Email" required="Don't forget">
     </div>
+    <div class="form-group">
+      <label id="number-label" for="number">Age <span class="clue">(optional)</span></label>
+      <input id="number-label" class="form-control" type="number"  name="age" min="1" max="50"  placeholoder="Age">
```
Comment (vitaliemiron) [inline app/index.html]: why age max 50??

File: app/scss/main.scss [hunk 4]
```diff
@@ -48,18 +48,19 @@ body {
 .form-control {
   display: block;
   width: 80%;
-  height: 100%;
-  padding: 5px;
+  height: 40%;
+  padding-bottom: 3px;
   color: #495057;
   background-color: #fff;
   background-clip: padding-box;
```
Comment (vitaliemiron) [inline app/scss/main.scss]: what this doing? Explain me why did you use this

File: app/scss/main.scss [hunk 5]
```diff
@@ -48,18 +48,19 @@ body {
...
+  border-radius: 3px;
 }
 
 .form-group {
-  margin-top: 0px;
-  margin-right: auto;
-  margin-bottom: 1.25rem;
-  margin-left: 90px;
-  padding: 0.25 rem;
+  margin-top: 0px autp 1.25rem;
```
Comment (vitaliemiron) [inline app/scss/main.scss]: autp ??

File: app/scss/main.scss [hunk 6]
```diff
@@ -48,18 +48,19 @@ body {
 .form-control {
   display: block;
   width: 80%;
```
Comment (vitaliemiron) [inline app/scss/main.scss]: remove width from this and set to your main container

File: app/index.html [hunk 7]
```diff
@@ -32,6 +32,11 @@ <h1 id="title" class="text-center main-title">freeCodeCamp Servey Form</h1>
       <label id="email-label" for="Email">Email</label>
       <input id="email" class="form-control" type="text" placeholder="Enter you Email" required="Don't forget">
     </div>
+    <div class="form-group">
+      <label id="number-label" for="number">Age <span class="clue">(optional)</span></label>
+      <input id="number-label" class="form-control" type="number"  name="age" min="1" max="100"  placeholoder="Age">
+    </div>
+  </form>  
   <link href="https://fonts.google.com/specimen/Roboto" rel="stylesheet" type="text/css">
```
Comment (vitaliemiron) [inline app/index.html]: font should be loaded in header

File: app/scss/main.scss (hunk 4 above)
Comment (veaceslavmocanu) [reply app/scss/main.scss]: I have change it

Comment (vitaliemiron) [reply app/scss/main.scss]: if i can see this code in github this means that you didn't change

File: app/scss/main.scss [hunk 8]
```diff
@@ -48,18 +48,19 @@ body {
...
-  height: 100%;
-  padding: 5px;
+  height: 40%;
```
Comment (vitaliemiron) [inline app/scss/main.scss]: use padding to set height not height selector

File: app/scss/main.scss (hunk 5 above)
Comment (vitaliemiron) [inline app/scss/main.scss]: fix syntax error, also for 0 you don't need to set px

**Pull request closed without merging**